.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+	extends: '@rowanmanning/eslint-config/legacy',
+	globals: {
+		define: true
+	},
+	rules: {
+		'id-length': 'off',
+		'no-void': 'off'
```

Comment:
We can avoid the use of void by adding a second parameter to the IIFE wrapper and using that as the safe `undefined`.

Reply:
Is `undefined` ever safe? Because it's set outside of a strict mode function

Reply:
Should be safe, yes:
```js
(function(module, undef) {
	console.log(module, undef);
}({}))
```

Reply:
Oooh! on it